packages/postgres-database/src/simcore_postgres_database/models/file_meta_data.py

```diff
@@ -22,6 +22,8 @@
     sa.Column("display_file_path", sa.String),
     sa.Column("created_at", sa.String),
     sa.Column("last_modified", sa.String),
-    sa.Column("file_size", sa.BigInteger)
-    #    sa.Column("state", sa.String())
+    sa.Column("file_size", sa.BigInteger),
+    # Entity tag (or ETag), represents a specific version of the object.
+    # SEE https://docs.aws.amazon.com/AmazonS3/latest/API/RESTCommonResponseHeaders.html)
+    sa.Column("entity_tag", sa.String),
```

Comment:
question: is this one not nullable?? it is nullable in the migration script.

Reply:
It has to be nullable because upon creation (at least right now) it that value is not known ... 
Let me explain. This is the sequence in [upload_link](https://github.com/ITISFoundation/osparc-simcore/blob/d39b77f0a8b4986318c261fad10e3d24f484e722/services/storage/src/simcore_service_storage/dsm.py) operation:
1. user request upload
2. entry in the file-metadata is created
3. task is fired to poll minio service about file uploaded
3. presigned link is returned
4. upload happens 
5. minio creates etag
6. task fired in 3 sets the value in database

Reply:
I'm ok with the fact that it can be nullable. it is just that the alembic migration scripts sets it as nullable. but the model you have here does not (line 28).